src/main/java/baseball/RandomNumber.java

```diff
@@ -0,0 +1,19 @@
+package baseball;
+
+import java.util.*;
+
+public class RandomNumber {
+    public List<String> generate() {
+        Random random = new Random();
+        List<String> result;
+        do {
+            result = List.of(String.valueOf(random.nextInt(999)).split(""));
+        } while (!checkDuplicate(result) || result.size() != 3);
+        return result;
+    }
+
+    private static boolean checkDuplicate(List<String> arr) {
```

Comment:
```suggestion
    private boolean checkDuplicate(List<String> arr) {
```
static 키워드는 제외해도 좋을 것 같습니다.